example/local/cog_example.py

```diff
@@ -0,0 +1,214 @@
+
+    @commands.command(name="play")
+    async def _play(self, ctx, music):
+        vc = self.Audio.getVC(ctx.guild.id)
+
+        if not vc:
+            return await ctx.send("Please type `!join` first.")
+
+        Song = await vc.loadSong(music)
+
+        if isinstance(Song, list):
+            return await ctx.send(
+                f"{len(Song) - 1} songs except {Song[0].title} added."
+            )
+        else:
+            return await ctx.send(f"{Song.title} added.")
+
+    @commands.command(name="skip")
+    async def _skip(self, ctx, offset: int = 1):
+        vc = self.Audio.getVC(ctx.guild.id)
+
+        if not vc:
+            return await ctx.send("Please type `!join` first.")
+
+        vc.skip(offset)
+
+        return await ctx.send(f"{offset} skipped.")
+
+    @commands.command(name="volume")
+    async def _volume(self, ctx, offset: int = 100):
+        vc = self.Audio.getVC(ctx.guild.id)
+
+        if not vc:
+            return await ctx.send("Please type `!join` first.")
+
+        Volume = vc.setVolume(offset / 100)
+
+        return await ctx.send(f"Set volume to {Volume * 100}%.")
+
+    @commands.command(name="crossfade")
+    async def _crossfade(self, ctx, offset: int = 10):
+        vc = self.Audio.getVC(ctx.guild.id)
+
+        if not vc:
+            return await ctx.send("Please type `!join` first.")
+
+        Crossfade = vc.setCrossfade(offset)
+
+        return await ctx.send(f"Set crossfade seconds to {Crossfade} seconds.")
+
+    @commands.command(name="autoplay")
+    async def _autoplay(self, ctx, offset: str = "on"):
+        vc = self.Audio.getVC(ctx.guild.id)
+
+        if not vc:
+            return await ctx.send("Please type `!join` first.")
+
+        offset = {"on": True, "off": False}.get(offset, True)
+
+        autoplay = vc.setAutoplay(offset)
+
+        return await ctx.send(
+            f'Auto related play {"enabled" if autoplay else "disabled"}.'
+        )
+
+    @commands.command(name="repeat")
+    async def _repeat(self, ctx, offset: str = "on"):
+        vc = self.Audio.getVC(ctx.guild.id)
+
+        if not vc:
+            return await ctx.send("Please type `!join` first.")
+
+        offset = {"on": True, "off": False}.get(offset, True)
+
+        repeat = vc.setRepeat(offset)
+
+        return await ctx.send(f'Repeat {"enabled" if repeat else "disabled"}.')
+
+    @commands.command(name="np")
+    async def _np(self, ctx):
+        vc = self.Audio.getVC(ctx.guild.id)
+
+        if not vc:
+            return await ctx.send("Please type `!join` first.")
+
+        return await ctx.send(
+            f"Now playing: {vc.player.current.title} `{vc.player.current.duration}:{vc.player.current.self.AudioData.duration}`"
+        )
+
+    @commands.command(name="shuffle")
+    async def _shuffle(self, ctx):
+        vc = self.Audio.getVC(ctx.guild.id)
+
+        if not vc:
+            return await ctx.send("Please type `!join` first.")
+
+        await vc.shuffle()
```

Comment:
Shuffle is not coroutine.

```suggestion
        vc.shuffle()
```